Gibe.AbTest/ExperimentCookieValue.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Gibe.AbTest
+{
+	public class ExperimentCookieValue
+	{
+		private readonly IAbTest _abTest;
+
+		private const string Seperator = "~";
+		private const string ExperimentSeperator = "-";
+
+		public string RawValue;
```

Comment:
Is this ever meant to be set externally?  If not, would it be better as:
```suggestion
		public string RawValue { get; private set; }
```